Subject: ppc64le remove vsx
From: Michel Normand <normand@fr.ibm.com>

temporarily remove the vsx related flags
as long as not supported for ppc64le
Note that also force as power4

Signed-off-by: Michel Normand <normand@fr.ibm.com>
Index: atlas/ATLAS/CONFIG/src/atlcomp.txt
===================================================================
--- atlas.orig/ATLAS/CONFIG/src/atlcomp.txt
+++ atlas/ATLAS/CONFIG/src/atlcomp.txt
@@ -187,9 +187,9 @@ MACH=PPCG5 OS=ALL LVL=1000 COMPS=dmc,icc
 MACH=PPCG5 OS=ALL LVL=1000 COMPS=skc
    'gcc' '-mpowerpc64 -maltivec -mabi=altivec -mcpu=970 -mtune=970 -O2 -mvrsave'
 MACH=POWER7 OS=ALL LVL=1010 COMPS=icc,smc,dmc,skc,dkc,xcc,gcc
-   'gcc' '-O2 -mvsx -mcpu=power7 -mtune=power7 -m64 -mvrsave -funroll-all-loops'
+   'gcc' '-O2 -m64 -mvrsave -funroll-all-loops'
 MACH=POWER7 OS=ALL LVL=1010 COMPS=f77
-   'gfortran' '-O2 -mvsx -mcpu=power7 -mtune=power7 -m64 -mvrsave -funroll-all-loops'
+   'gfortran' '-O2 -m64 -mvrsave -funroll-all-loops'
 MACH=POWER6 OS=ALL LVL=1010 COMPS=icc,smc,dmc,skc,dkc,xcc,gcc
    'gcc' '-mcpu=power6 -mtune=power6 -maltivec -O3 -fno-schedule-insns -fschedule-insns2 -minsert-sched-nops=2'
 MACH=POWER5 OS=ALL LVL=1010 COMPS=icc,smc,dmc,skc,dkc,xcc,gcc
Index: atlas/ATLAS/CONFIG/src/probe_comp.c
===================================================================
--- atlas.orig/ATLAS/CONFIG/src/probe_comp.c
+++ atlas/ATLAS/CONFIG/src/probe_comp.c
@@ -446,8 +446,8 @@ COMPNODE **GetDefaultComps(enum OSTYPE O
 
    if ((vecexts & (1<<ISA_AVXFMA4)) && arch == AmdDozer)
       vp = "-msse4.2 -mfma4";
-   else if (vecexts & (1<<ISA_VSX))
-      vp = "-mvsx";
+   /*else if (vecexts & (1<<ISA_VSX))
+      vp = "-mvsx";*/
    else if (vecexts & (1<<ISA_AV))
       vp = "-maltivec";
    else if (vecexts & (1<<ISA_AVX))
